enterprise/backend/src/metabase_enterprise/sso/api/sso.clj

```diff
@@ -22,7 +22,8 @@
          metabase-enterprise.sso.integrations.saml/keep-me)
 
 (defn- throw-if-no-premium-features-token []
```

Comment:
nit: may be we could replace this with the new `assert-has-feature`

Reply:
Done. I moved the token checking to the implementations of `sso-get` and `sso-post` so that the error message could include the specific backend being used (JWT vs SSO)